app/models/credentials.rb

```diff
@@ -63,8 +70,7 @@ def validate
 
     validates_presence [ :api_key ]
 
-    errors.add(:password, 'must not be blank') if @plain_password && @plain_password.empty?
```

Comment:
Are we relaying on "must no be blank" or "cannot contain a newline" somewhere? just like we are now with ":insufficient_password_complexity"